Small fix to pool_image_uri()

johngrimsey opened the pull request:
If the passed object to pool_image_uri() has no image objects assigned,
nothing was output before. Now, if the placeholders option is true, a
placeholder will be output.

Commit:
Placeholder can be output when no sfImagePoolImage found

If the passed object to pool_image_uri() has no image objects assigned,
nothing was output before. Now, if the placeholders option is true, a
placeholder will be output.

angelsk commented:
This should also respect the placehold.it setting as per /lib/helper/ImagePoolHelper.php#L209

Commit:
placehold.it can now be used with no sfImagePoolImage associated

If placehold.it is enabled, it takes precedence over using
placeholder.jpg (as usual).
